src/casp/data_manager/base_data_manager.py

```diff
@@ -0,0 +1,27 @@
+from google.cloud import bigquery
+
+from casp.services import utils
+from casp.services.db import init_db
+
+
+class BaseDataManager:
+    """
+    Base class for all Data Managers.  It also provides a BigQuery client and a Postgres DB session.
+    Data Managers are used by services to communicate with databases.
+
+    Class attributes:
+    credentials: Google service credentials object
+    project: Google Cloud project ID
+    bigquery_client: Google BigQuery client object to communicate with BigQuery
+    postgres_db_session: Postgres DB session object to communicate with Postgres DB
+    SQL_TEMPLATE_DIR (has to be set): Path to a directory with SQL templates related to Cellarium API.
+    """
+
+    SQL_TEMPLATE_DIR: str
+
+    def __init__(self):
+        credentials, project = utils.get_google_service_credentials()
+        self.credentials = credentials
+        self.project = project
+        self.bigquery_client = bigquery.Client(credentials=self.credentials, project=self.project)
+        self.postgres_db_session = init_db()
```

Comment:
How about renaming `postgres_db_session` to `usage_tracking_db_session` or something like that?

Reply:
The term 'usage_tracking_db_session' doesn't really appeal to me, as we are planning to incorporate more data models into this database. Once we implement cell metadata model in postgres, this name will become outdated, and that change will happen very soon. I believe we should decide on names for these two databases first, which will simplify the process of naming their sessions and clients. Here are my suggestions:

For the PostgreSQL database:

1. 'System Info DB' with the session named 'system_info_db_session'
2. 'System Data DB' with the session named 'system_data_db_session'

For BigQuery, considering that it seems to be used exclusively for storing raw count matrices, we could name it something like 'MatrixDataBase', 'Sparse Matrix DB', or if it is too specific, smth like 'Bigdata DB'?

Reply:
Sounds all good to me. For BQ, the most accurate name is `BlockCOOMatrixDatabase`.